README.md

```diff
@@ -30,125 +29,87 @@ We are working under an agile framework, and plan to use this repo to publish, r
-	```
-	(OAH)username$
-	```
-
-	If the virtualenv did not activate right away, run this command from the directory above where the OAH directory lives:
-	```
-	$ source OAH/bin/activate
-	```
-
-	To deactivate, use the command `deactivate`.
-
-4. Install sheer into the virtualenv with the `-e` flag (which allows you to make changes to sheer itself). The path to sheer is the root directory of the GitHub repository you checked out (cloned) earlier, which likely will be `./sheer`:
-	```
-	$ pip install -e ~/path/to/sheer
-	```
-
-5. Install sheer's python requirements:
-	```
-	$ pip install -r ~/path/to/sheer/requirements.txt
-	```
-
-6. You should now be able to run the sheer command:
-	```
-	$ sheer
-	usage: sheer [-h] [--debug] {inspect,index,serve} …
-	sheer: error: too few arguments
-	```
-
-7. Populate Elasticsearch with content:
-  ```
-  $ [WORDPRESS=http://wordpress.uri] sheer index [--reindex]
-  ```
-  Make sure to be in the site's root folder and that Elasticsearch is running. The above
-  command will read all OaH content and save it in Elasticseach. [More information](#wordpress-indexing).
-
-If you run into problems or have any questions about Sheer, check out [Sheer on Github](https://github.com/cfpb/sheer) and the [Sheer Issue Tracker](https://github.com/cfpb/sheer/issues).
 
 ### Node, Grunt, Bower, Browserify
 
 1. Install [node.js](http://nodejs.org/), either v0.12 or io.js. If you're using the default version that comes with Mac OSX, you'll need to upgrade - first [install NVM](https://github.com/creationix/nvm).
 1. If you installed with NVM, then you can install and set io.js as your default node version:
-	```
-	$ nvm install iojs
-	$ nvm alias default iojs
+	```sh
+	nvm install iojs
+	nvm alias default iojs
 	```
 2. Install [Grunt](http://gruntjs.com/), [Bower](http://bower.io/) and [Browserify](http://browserify.org/):
-	```
-	$ npm install -g grunt-cli bower browserify
+	```sh
+	npm install -g grunt-cli bower browserify
 	```
 
 3. Navigate to the cloned `owning-a-home` directory and install the project's node dependencies:
-	```
-	$ npm install
+	```sh
+	npm install
 	```
 
 4. Navigate to the `config` folder. In that folder, copy the `example-config.json` file and rename it `config.json`. This can be done from the command line with the following two commands:
-	```
-	$ cd config
-	$ cp example-config.json config.json
+	```sh
+	cd config
+	cp example-config.json config.json
 	```
 
 5. Run grunt to build the site:
-	```
-	$ grunt
+	```sh
+	grunt
 	```
 
 [npm-shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap) is used to lock down dependencies. If you add any dependencies to package.json, re-run `npm shrinkwrap` to generate a new `npm-shrinkwrap.json` file.
 
+### cfgov-refresh
+
+Owning a Home can be installed and run by the cfgov-refresh app.
```

Comment:
"Must be", right? If we're dropping support for running it with standalone Sheer?